docs/conf.py

```diff
@@ -65,22 +65,10 @@
 #
 
 html_show_sourcelink = True
-html_sidebars = {
-    "**":
-        [
-            "logo-text.html", "globaltoc.html", "localtoc.html",
-            "searchbox.html"
-        ]
-}
 
-html_theme = 'sphinx_material'
+html_theme = 'sphinx_symbiflow_theme'
+
 html_theme_options = {
-    'nav_title': project,
-    'color_primary': 'deep-purple',
-    'color_accent': 'purple',
-    'repo_name': "symbiflow/symbiflow-examples",
-    'repo_url': 'https://github.com/symbiflow/symbiflow-examples',
```

Comment:
You probably want to keep this - https://github.com/google/skywater-pdk/pull/341/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3R165

Reply:
this doesn't seem to be supported in the symbiflow theme. The build says:
```
WARNING: unsupported theme option 'repo_url' given
```

this combined with https://github.com/SymbiFlow/symbiflow-examples/blob/master/.readthedocs.yml#L26 fails on RTD.

I suppose we should now use `github_url` (according to https://sphinx-symbiflow-theme.readthedocs.io/en/latest/customization.html)

Reply:
I added `github_url`